configs/commercial.js

```diff
@@ -1,10 +1,14 @@
-const path = require('path');
+import licenseHeaderPlugin from 'eslint-plugin-license-header';
 
-module.exports = {
-  plugins: [
-    'license-header'
-  ],
-  rules: {
-    'license-header/header': [2, path.join(__dirname, '/../resources/commercial-license-header.js') ]
+import commercialLicenseHeader from '../resources/commercial-license-header.js'
```

Comment:
Retrieve a file path, local from the current file, in an ES module:

```
const { fileURLToPath } = require('node:url');

const path = fileURLToPath(new URL('somePath', import.meta.url));
```

Cf. [URL](https://nodejs.org/api/url.html) for tricks available.

Reply:
✅